src/xdocs/checks/naming/abbreviationaswordinname.xml

```diff
@@ -159,232 +158,233 @@
+  final int customerID = 2;
+  static int nextID = 3;
+  // violation below 'no more than '1' consecutive capital letters'
+  static final int MAX_ALLOWED = 4;
+</source>
         <p>
           To configure to check variables, enforce
           no abbreviations (essentially camel case) except for
           words like 'ALLOWED'.
         </p>
-        <p>Configuration:</p>
+        <p id="Example7-config">Configuration:</p>
+        
+          
+          
         <source>
-&lt;module name="AbbreviationAsWordInName"&gt;
-    &lt;property name="allowedAbbreviations" value="ALLOWED"/&gt;
-    &lt;property name="ignoreStaticFinal" value="false"/&gt;
-&lt;/module&gt;
-        </source>
-        <p>Example:</p>
-        <source>
-public class MyClass {
-    public int counterXYZ = 1;                // OK
-    public final int customerID = 2;          // OK
-    public static int nextID = 3;             // OK
-    public static final int MAX_ALLOWED = 4;  // OK, abbreviation is allowed
-}
-        </source>
-      </subsection>
-
-      <subsection name="Example of Usage" id="Example_of_Usage">
-        <ul>
-          <li>
-            <a href="https://github.com/search?q=path%3Asrc%2Fmain%2Fresources%20path%3A**%2Fgoogle_checks.xml+repo%3Acheckstyle%2Fcheckstyle+AbbreviationAsWordInName">
-            Google Style</a>
-          </li>
-          <li>
-            <a href="https://github.com/search?q=path%3Aconfig%20path%3A**%2Fcheckstyle-checks.xml+repo%3Acheckstyle%2Fcheckstyle+AbbreviationAsWordInName">
-            Checkstyle Style</a>
-          </li>
-        </ul>
-      </subsection>
-
-      <subsection name="Violation Messages" id="Violation_Messages">
-        <ul>
-          <li>
-            <a href="https://github.com/search?q=path%3Asrc%2Fmain%2Fresources%2Fcom%2Fpuppycrawl%2Ftools%2Fcheckstyle%2Fchecks%2Fnaming%20path%3A**%2Fmessages*.properties+repo%3Acheckstyle%2Fcheckstyle+%22abbreviation.as.word%22">
-            abbreviation.as.word</a>
-          </li>
-        </ul>
-        <p>
-          All messages can be customized if the default message doesn't suit you.
-          Please <a href="../../config.html#Custom_messages">see the documentation</a>
-          to learn how to.
-        </p>
-      </subsection>
-
-      <subsection name="Package" id="Package">
-        <p> com.puppycrawl.tools.checkstyle.checks.naming </p>
-      </subsection>
-
-      <subsection name="Parent Module" id="Parent_Module">
-        <p> <a href="../../config.html#TreeWalker">TreeWalker</a> </p>
-      </subsection>
-    </section>
-  </body>
-</document>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;AbbreviationAsWordInName&quot;&gt;
+      &lt;property name=&quot;allowedAbbreviations&quot; value=&quot;ALLOWED&quot;/&gt;
+      &lt;property name=&quot;ignoreStaticFinal&quot; value=&quot;false&quot;/&gt;
+    &lt;/module&
```

Comment:
The file generation stops here and no errors are thrown. I believe I am hitting some buffer size limit because the file contains exactly 16384 characters (2^14). Could it be?

![Screenshot from 2023-07-11 14-25-12](https://github.com/checkstyle/checkstyle/assets/23459549/9ed6b877-c136-4aaf-a7a2-4b3001ff9bdb)

Reply:
Resolved. This was the issue https://github.com/apache/maven-doxia/blob/24ab8ba193ec44007333775576277f9aa9ea165f/doxia-modules/doxia-module-xdoc/src/main/java/org/apache/maven/doxia/module/xdoc/XdocSink.java#L249. I had to call flush() myself.

    flush();